modules/command-engine/scaladsl/src/main/scala/surge/scaladsl/common/HealthCheckTrait.scala

```diff
@@ -9,4 +9,5 @@ import scala.concurrent.Future
 
 trait HealthCheckTrait {
   def healthCheck: Future[HealthCheck]
+  def ready: Future[HealthCheck]
```

Comment:
Maybe `readiness` on this as well for naming

Reply:
done